OWNERS_ALIASES

```diff
@@ -25,12 +25,6 @@ aliases:
   - varshaprasad96
   - inteon
 
-  # folks to can approve things in the directly-ported
```

Comment:
@vincepri Should we get rid of the OWNERS file as well?

Doesn't make much sense if we don't use it